addons/website/models/website.py

```diff
@@ -2066,8 +2067,49 @@ def _trigram_enumerate_words(self, search_details, search, limit):
         :param limit: maximum number of records fetched per model to build the word list
         :return: yields words
         """
+        def get_subquery(model, fields, id_column, rel_table='', rel_joinkey=''):
```

Comment:
`get_subquery` doesn't say much (it's quite generic). Do you have an idea for a better name?
Or either then it could get a little docstring to explain the query it's supposed to generate. It's just not obvious what this is doing.

Reply:
I renamed it slightly and added a docstring